code/include/observations.h

```diff
@@ -47,14 +47,16 @@ Motivated by investigation into the monocular case of:
 #include "camera_stream.h"
 #include <opencv2/opencv.hpp>
 
+#define TICKS_PER_SEC 1000000000l
+#define TIME_TYPE std::chrono::duration<long int, std::ratio<1l, TICKS_PER_SEC> >
+#define CLOCK_TYPE std::chrono::time_point<std::chrono::_V2::steady_clock, std::chrono::duration<long int, std::ratio<1l, TICKS_PER_SEC> > >
```

Comment:
Can be replaced with

~~~~~
#define CLOCK_TYPE std::chrono::time_point<std::chrono::steady_clock>
~~~~~